apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html

```diff
@@ -91,13 +91,13 @@
 
 <ng-container *ngIf="(isGetPurchaseDetailsSuccess$ | async)">
     <div class="clearfix sales-register-table">
-        <div class="responsive mt-2">
-            <table class="table basic table-bordered table-middle ">
+        <div class="responsive table-responsive">
+            <table class="table basic giddh-table onMobileView">
```

Comment:
onMobileView class should be in small letters and separated by - instead of camel capsing

Reply:
we are using for mobile responsive **onMobileView** class in all component